### PR TITLE
allow to omit specifying clock and reset signals

### PR DESCRIPTION
**Pull request by taichi-ishitani**
resolve veryl-lang/veryl#569

### CODE REVIEW

File: crates/analyzer/src/handlers/create_symbol_table.rs [hunk 1]
```diff
@@ -197,7 +272,14 @@ impl<'a> VerylGrammarTrait for CreateSymbolTable<'a> {
                 affiniation,
             };
             let kind = SymbolKind::Variable(property);
-            self.insert_symbol(&arg.identifier.identifier_token.token, kind, false);
+            let id =
+                self.insert_symbol(&arg.identifier.identifier_token.token, kind.clone(), false);
+
+            if self.is_default_clock_candidate(kind.clone()) {
+                self.default_clock_candidates.push(id.unwrap());
```
Comment (dalance) [inline crates/analyzer/src/handlers/create_symbol_table.rs:279]: Could you handle the case of `id` is `None`?
Probably "duplicated identifier" causes panic at `id.unwrap()`.

Comment (taichi-ishitani) [reply crates/analyzer/src/handlers/create_symbol_table.rs:279]: My understanding is that "duplicated identidifier" will be checked here.
https://github.com/veryl-lang/veryl/blob/c2fcbe9d35cd7e839eb79cefeb4ad2e2fba843fe/crates/analyzer/src/handlers/create_symbol_table.rs#L96-L103

Therefore, I think we can handle `id` as a `Some` object.

Comment (dalance) [reply crates/analyzer/src/handlers/create_symbol_table.rs:279]: If `id` is `None`, the error are registerd by `self.errors.push` and the `id` is returned normally.
So we can't assume `id` is always `Some`.

Comment (taichi-ishitani) [reply crates/analyzer/src/handlers/create_symbol_table.rs:279]: I understand.
I will fix it.

Comment (taichi-ishitani) [reply crates/analyzer/src/handlers/create_symbol_table.rs]: I've fixed your feedback.

**Comment by dalance** [comment]
Looks good.
Thanks!

**Pull request merged**